### ISSUE TITLE
Estudar formas para questionário digital (on-line e/ou offline)

### ISSUE THREAD

**Issue by lucasmation** [lucasmation/aval_Brasil_mais_Prod#32]
Na reunião de 13/abr o SENAI mostrou abertura para um sistema digital de coleta de informações. Mas disseram que seria melhor algo offline, baseado em laptop. 

@damatadaniel , precisamos definir uma pessoa esperta para estudar esta questão na semana que vem. 
Talvez a Fernanda, bolsista do laboratório seja um boa pessoa para isso, com ajuda do Pedro A.

**Comment by damatadaniel** [comment]
Concordo plenamente. Vamos perguntar para os SENAIs locais se há abertura para isso.

________________________________

**Comment by JuliaAkaishi** [comment]
@lucasmation  e @damatadaniel 

Para uma plataforma off-line, podemos adaptar o excel.

**Comment by lucasmation** [comment]
Pode ser. Mas acho que precisamos dar uma  olhada com calma nas  opções. Pedia ajuda da Fernanda (do IpeaDATA-lab) nisso, para não atrapalhar nosso trabalho nas pendências urgentes.

@JuliaAkaishi , acho que uma primeira tarefa é listar aqui as funcionalidades que o questionário precisa ter. 
Por exemplo: 

- "Pular perguntas" em função de respostas a perguntas anteriores. 

o que mais?

**Comment by JuliaAkaishi** [comment]
@lucasmation 

De funcionalidades, acho que iríamos precisar de

- Possibilidade de questões múltipla escolha;
- Possibilidade de criar questões abertas;
- Possibilidade de criar respostas em escala (para a parte de avaliação do consultor)

Alguns pontos Lucas: 

- Pelo que eu entendi o pessoal do GT não apresentou postura de querer cortar as partes do questionário. Acho que poderíamos conversar sobre a possibilidade de tirar a 8 (Qualidade).
- O Fábio comentou sobre o acesso a linhas de crédito para empresas inovadoras, e que empresas que fizeram parte do B+P conseguiram. Acho que poderíamos adicionar aquela seção de crédito e financiamento do Q2. O que você acha?

**Comment by lucasmation** [comment]
Documentando aqui o que temos falado por email: 

### Sugestões no site do JPAL

- [measurement-and-data-collection](https://www.povertyactionlab.org/research-resources/measurement-and-data-collection) > seção "data collection"


### Sugestões do Steve

SurveyCTO is what all the JPAL /IPA people use

### Sugestões da Lycia:

- https://www.questionpro.com: parece funcionar offline, poderia ser uma solução. 
- surveymonkey​: é muito bom. Temos até instalado no Ipea. Premite mandar emails com links para responder questionários que associam (pelo link enviado) a resposta ao e-mail enviado. 
Mas pelo que olhei não permite offline. 
- googleforms : fácil de fazer mas travado e não funciona offline

### Sugestões Stack economics.stackexchange.com

- fiz [esta pergunta aqui](https://economics.stackexchange.com/questions/21898/software-for-field-work-collecting-survey-data-multi-platform-on-line-and-off): já há algumas respostas

**Comment by lucasmation** [comment]
Budget: não se preocupem c isso, QQ coisa até 400usd /ano tá valendo

Em qua, 9 de mai de 2018 13:24, JuliaAkaishi <notifications@github.com>
escreveu:

**Comment by JuliaAkaishi** [comment]
@lucasmation 

qual plataforma vamos usar?

**Comment by JuliaAkaishi** [comment]
1) E as perguntas tabulares, quais tem capacidade de fazer?
- Surveymonkey: sim
- SurveyCTO: sim (http://blog.surveycto.com/features-news-html-tables-gps/)
- brewsurvey: não
- Questionpro: não
- Pendragon: não

2)tem certeza que o SurveyCTO não tem capacidade de pular pergunta?
Não foi mencionado que o SurveyCTO não é capaz de pular questão. Por ter como base o ODK ele tem sim essa funcionalidade, mas **funciona somente em android**,
